tequilapi/client/client.go

```diff
@@ -100,13 +102,17 @@ func (client *Client) RegistrationStatus(address string) (RegistrationStatusDTO,
 }
 
 // Connect initiates a new connection to a host identified by providerID
-func (client *Client) Connect(consumerID, providerID string) (status StatusDTO, err error) {
+func (client *Client) Connect(consumerID, providerID string, disableKill bool) (status StatusDTO, err error) {
+	connectOptions := dto.ConnectOptions{DisableKillSwitch: disableKill}
+
 	payload := struct {
-		Identity   string `json:"consumerId"`
-		ProviderID string `json:"providerId"`
+		Identity       string             `json:"consumerId"`
+		ProviderID     string             `json:"providerId"`
+		ConnectOptions dto.ConnectOptions `json:"connectOptions"`
```

Comment:
It's dont have to repeat `connectOptions` in `connectionRequest`. IMHO `options` would be enought in REST contract